src/components/ViewRoutingSlip/PaymentInformation.vue

```diff
@@ -144,4 +185,10 @@ export default class PaymentInformation extends Vue {
 .pay-info .col {
   padding-left: 0px;
 }
+.button-margin {
+  margin-top: 0.5em;
```

Comment:
mt-2 in the class should be equivalent? 

https://vuetifyjs.com/en/styles/spacing/